lib/github.js

```diff
@@ -183,9 +183,7 @@ class GitHub extends Release {
         debug(options);
         const response = await this.client.repos.updateRelease(options);
         debug(response.data);
-        const { html_url } = response.data;
```

Comment:
This is where the updated release URL would come out, this is why this override exists...

Reply:
*existed. This got removed by the patch. To we need that, but we also need that to play nice with the stubs. 🤔

Reply:
OK now I get it. We can send the `tag_name` again with the `options` in the `repos.updateRelease` (so it will also come back with the stub response) and restore the line I removed here. Apologies for the confusion!

Reply:
Well, this should be it...